tests/python/pants_test/backend/project_info/tasks/test_export_dep_as_jar.py

```diff
@@ -611,3 +625,14 @@ def test_imports_3rdparty_jars_from_transitive_dependencies(self):
       'org.apache:apache-jar:12.12.2012',
       result['targets'][spec]['libraries']
     )
+
+  def test_libraries_respect_strict_deps(self):
+    enabled_spec = self.strict_deps_enabled.address.spec
+    enabled_result = self.execute_export_json(enabled_spec)['targets'][enabled_spec]
+    disabled_spec = self.strict_deps_disabled.address.spec
+    disabled_result = self.execute_export_json(disabled_spec)['targets'][disabled_spec]
+
+    dependency_library_entry = self.jvm_target_with_sources.address.spec.replace(":", ".")
```

Comment:
Just a suggestion: it might be good to add a brief comment here saying `jvm_target_with_sources` is a transitive dep of `strict_deps_enabled` and `strict_deps_disabled`, so the assertions would be more obvious.

Alternatively we could also rename `dependency_library_entry` to `transitive_ dependency_library_entry`

Reply:
Thanks for the suggestion! Made the change.